pkg/agent/netpol/netpol.go

```diff
@@ -131,8 +133,27 @@ func Run(ctx context.Context, nodeConfig *config.Node) error {
 	npInformer.AddEventHandler(npc.NetworkPolicyEventHandler)
 
 	wg.Add(1)
-	logrus.Info("Starting the netpol controller")
+	logrus.Infof("Starting the netpol controller version %s", getKubeRouterVersion())
```

Comment:
1. Set the correct version at build-time in https://github.com/k3s-io/k3s/blob/d5ffc4b1d10813d98918c2560d6c82f90bc0c16c/scripts/build#L20-L44

2. Add an import for `github.com/cloudnativelabs/kube-router/pkg/version` and then do:
```suggestion
	logrus.Infof("Starting the netpol controller version %s", version.Version)
```

Reply:
You might also make the message look like the one upstream at https://github.com/k3s-io/kube-router/blob/master/pkg/version/version.go#L16:

`fmt.Sprintf("Running %v version %s, built on %s, %s\n", "kube-router", Version, BuildDate, runtime.Version())

Reply:
I changed it as you asked, now it should display a line like this:
```
INFO[0004] Starting the netpol controller version v1.5.2-0.20221026101626-e01045262706, built on 2022-11-03T11:19:44Z, go1.19.2
```